src/main/java/org/mockito/verification/Timeout.java

```diff
@@ -63,4 +66,9 @@ public VerificationMode never() {
     public VerificationMode description(String description) {
         return VerificationModeFactory.description(this, description);
     }
+
+    @Override
+    public void verifyInOrder(VerificationDataInOrder data) {
+        System.out.println(data);
```

Comment:
That was an accident. I've removed it in the latest commit. Haven't delivered it yet. This was leftover with testing to see if I could get the refactoring working without doing any wrapping class weirdness.